test/apps/gups.c

```diff
@@ -321,10 +321,10 @@ SHMEMRandomAccess(void)
                        * NumUpdates_Default due to execution time bounds */
   int64_t ProcNumUpdates; /* number of updates per processor */
 
-  static long llpSync[_SHMEM_BCAST_SYNC_SIZE];
+  static long pSync_bcast[_SHMEM_BCAST_SYNC_SIZE];
```

Comment:
Could you remove the leading ```_``` from the constants?  The ```_SHMEM_*``` constants are deprecated and were replaced by ```SHMEM_*``` versions.